test/e2e/e2e.go

```diff
@@ -26,15 +26,14 @@ import (
 
 // Setup creates the client objects needed in the e2e tests.
 func Setup(t *testing.T) *test.Clients {
-	return SetupWithNamespace(t, test.ServingOperatorNamespace)
+	return SetupWithNamespace(t)
```

Comment:
Could you consolidate the function SetupWithNamespace with Setup?

Reply:
Good call. Done.